ui/component/notification/index.js

```diff
@@ -1,8 +1,19 @@
 import { connect } from 'react-redux';
```

Comment:
![image](https://user-images.githubusercontent.com/64950861/127956589-d1cb0211-911a-4c04-89cd-1944030cb047.png)

- [ ] When I click on this kind of notification, I'll go to the intended page, but it immediately sends me back to the Notification page.